File: create_veterinary/config/dbconfig.php
<?php
class DatabaseConnector {
    private $dsn = "pgsql:host=postgresql-karen.alwaysdata.net;port=5432;dbname=karen_dbveterinary;user=karen_programacion;password=KPCA0207-";

    public function connect() {
        $conn = null;  
        try {
            $conn = new PDO($this->dsn);
            $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
            echo "Conexión establecida.";
        } catch (PDOException $e) {
            echo "Error al conectar: " . $e->getMessage();
        }
        return $conn;
    }
}
?>

File: update_medicines/config/dbconfig.py
import psycopg2

class DatabaseConnector:
    def connect(self):
        """Intenta establecer una conexión con la base de datos PostgreSQL."""
        try:
            conn = psycopg2.connect(
                dbname='karenchicaiza_dbmedicines',
                user='karenchicaiza_programacion',
                password='karenpat0209-',
                host='postgresql-karenchicaiza.alwaysdata.net',
                port='5432'
            )
            print("Conexión establecida.")
            return conn
        except Exception as e:
            print("Error al conectar:", e)
            return None
            
        